packages/gas-fee-controller/CHANGELOG.md

```diff
@@ -5,8 +5,13 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+## [10.0.1]
 ### Changed
+- **BREAKING:** Bump dependency and peer dependency on `@metamask/network-controller` to ^16.0.0
+- Bump @metamask/auto-changelog from 3.4.2 to 3.4.3 ([#1997](https://github.com/MetaMask/core/pull/1997))
```

Comment:
The `auto-changelog` entries can all be removed. It's a dev tool, it doesn't affect the package at all, so it's not relevant for the audience of "users of this package". That's who these changelogs are written for.

Reply:
Fixed https://github.com/MetaMask/core/pull/2044/commits/c31381478dbbdcc435ee93b75f2b0dee32680202!